plugins/org.jkiss.dbeaver.cmp.simple/src/org/jkiss/dbeaver/tools/compare/simple/CompareReportRenderer.java

```diff
@@ -106,10 +106,15 @@ private void renderHeader() throws IOException
         xml.startElement("style");
         StringBuilder styles = new StringBuilder();
         styles.append("table {font-family:\"Lucida Sans Unicode\", \"Lucida Grande\", Sans-Serif;font-size:12px;text-align:left;} ");
-        styles.append(".missing {color:red;} .differs {color:red;} ");
+        //styles.append(".missing {color:red;} .differs {color:red;} ");
+        styles.append(".missing {color:blue;} .differs {color:blue;}"); //visibility!
```

Comment:
Let's use different colors for missing and differs. We suppose that missing should be red and differs may stay blue.

Reply:
Ready